src/sentry/replays/endpoints/organization_replay_selector_index.py

```diff
@@ -0,0 +1,214 @@
+        query_options["offset"] = Offset(pagination.offset)
+
+    sorting = get_valid_sort_commands(
+        sort,
+        default=OrderBy(Column("count_dead_clicks"), Direction.DESC),
+        query_config=SelectorQueryConfig(),
+    )
+
+    snuba_request = SnubaRequest(
+        dataset="replays",
+        app_id="replay-backend-web",
+        query=Query(
+            match=Entity("replays"),
+            select=[
+                Column("click_tag"),
+                Column("click_id"),
+                Column("click_class"),
+                Column("click_text"),
+                Column("click_role"),
+                Column("click_alt"),
+                Column("click_testid"),
+                Column("click_aria_label"),
+                Column("click_title"),
+                Function("sum", parameters=[Column("click_is_dead")], alias="count_dead_clicks"),
+                Function("sum", parameters=[Column("click_is_rage")], alias="count_rage_clicks"),
+            ],
+            where=[
+                Condition(Column("project_id"), Op.IN, project_ids),
+                Condition(Column("timestamp"), Op.LT, end),
+                Condition(Column("timestamp"), Op.GTE, start),
+                Condition(Column("click_tag"), Op.NEQ, ""),
+            ],
+            orderby=sorting,
+            groupby=[
+                Column("click_tag"),
+                Column("click_id"),
+                Column("click_class"),
+                Column("click_text"),
+                Column("click_role"),
+                Column("click_alt"),
+                Column("click_testid"),
+                Column("click_aria_label"),
+                Column("click_title"),
+            ],
+            granularity=Granularity(3600),
+            **query_options,
+        ),
+        tenant_ids=tenant_ids,
+    )
+    return raw_snql_query(snuba_request, "replays.query.query_replays_dataset")
+
+
+def process_raw_response(response: list[dict[str, Any]]) -> list[dict[str, Any]]:
+    """Process the response further into the expected output."""
+
+    def _iter_response():
+        for r in response:
+            # Tag is always present.
+            selector = r["click_tag"]
+
+            if r["click_id"]:
+                selector = selector + f"#{r['click_id']}"
+            if r["click_class"]:
+                selector = selector + "." + ".".join(r["click_class"])
+
+            if r["click_role"]:
+                selector = selector + f'[role="{r["click_role"]}"]'
+            if r["click_alt"]:
+                selector = selector + f'[alt="{r["click_alt"]}"]'
+            if r["click_testid"]:
+                selector = selector + f'[testid="{r["click_testid"]}"]'
+            if r["click_aria_label"]:
+                selector = selector + f'[aria="{r["click_aria_label"]}"]'
+            if r["click_title"]:
+                selector = selector + f'[title="{r["click_title"]}"]'
+
+            yield {
```

Comment:
i'm not as familiar with some python conventions -- is there any advantage to yielding if code is synchronous? why not just return a list directly?